app/models/user.rb

```diff
@@ -0,0 +1,31 @@
+require 'bcrypt'
+require_relative './peep'
+require_relative './reply'
+
+class User
+
+  attr_reader :password
```

Comment:
Why do you need attribute readers/accessors for password and password_confirmation?